wifiphisher/common/interfaces.py

```diff
@@ -623,13 +623,79 @@ def on_exit(self):
                 mac_address = adapter.original_mac_address
                 self.set_interface_mac(interface, mac_address)
 
+        if (hasattr(power_network_manager, 'has_disable_nm') and
+                power_network_manager.has_disable_nm):
+            # recover the network manager if we have disabled it past
```

Comment:
Yes, `toggle_networking` is better since it involves both enabling and disabling of the function.